.github/workflows/build_test_linux_fedora.yaml

```diff
@@ -200,6 +200,12 @@ jobs:
     - name: Run Binaries 2/2
       run: ./build/dpsim/examples/cxx/WSCC_9bus_mult_decoupled
 
+    - name: Run Binaries 2/2
+      run: ./build/dpsim/examples/cxx/WSCC_9bus_split_decoupled
+
+    - name: Run Binaries 2/2
+      run: ./build/dpsim/examples/cxx/WSCC_9bus_split_decoupled_EMT
+
```

Comment:
@leonardocarreras should we make it also part of [build_test_linux_rocky_profiling](https://github.com/sogno-platform/dpsim/blob/master/.github/workflows/build_test_linux_rocky_profiling.yaml)?

Reply:
It could be included here too (sounds like a nice suggestion), I would not consider it as blocking issue as we are not running it on every push request https://github.com/sogno-platform/dpsim/blob/d66f8267f1abed95a379ec2baf5df8e7131f5d98/.github/workflows/build_test_linux_rocky_profiling.yaml#L40-L54

Reply:
Added examples to the workflow